app/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
     id("com.android.application")
     id("org.jetbrains.kotlin.android")
+    kotlin("kapt")
```

Comment:
I noticed that the way this was done is the assignment was with `ksp`. Just wanted to flag this in case we run into any issues with Room in the future.

Reply:
Sure i can look into it, from what I had in my hw5 I had used kapt.

Reply:
Nevermind, you're right--`kapt` was used in the starter code.